### ISSUE TITLE
Tensorboard loss is zero

### ISSUE THREAD

**Issue by aranguri** [allenai/bi-att-flow#112]
I'm running the dev branch on tensorflow 1.3.0 and python 3.5.2. The loss at tensorboard is zero throughout the training. Should the tensorboard log work correctly?

I'm running the code with the command
> python -m basic.cli --mode train --noload --len_opt --cluster

is there a way to solve this?

Thanks :)
